### PR TITLE
Fixing test

### PR DESCRIPTION
**Pull request by lanre-ade**

### CODE REVIEW

File: content_api/test_content_api.py [hunk 1]
```diff
@@ -391,68 +393,65 @@ def test_get_post_with_comments(self):
...
+    #
+    #     s.pretty_print(response.data)
+    #     self.assertEquals(len(response.data), 4)
+    #
+    #     authors = [
+    #         self.author_a,
+    #         self.author_b,
+    #         self.author_c,
+    #         author ]
+    #
```
Comment (xkb) [inline content_api/test_content_api.py]: lol?

Comment (lanre-ade) [inline content_api/test_content_api.py]: Hahaha, I'll fix it up for real in a bit

**Comment by lanre-ade** [comment]
:wave:

**Comment by mraypold** [comment]
I think this is good to go now. Friend and FoF are now integrated.

**Pull request merged**